impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js

```diff
@@ -60,8 +60,6 @@ define([
             plotFrameVisible: false,
             plotSizeMin: {width: 600, height: 300},
```

Comment:
yeah, but you can override this. also if we think that this shouldn't be applied by default because of, e.g. CDF, we can also move this to container specific configs